### ISSUE TITLE
[BUG] Changing random seed has no effect when fine tuning Chronos

### ISSUE THREAD

**Issue by ziyulu-uw** [autogluon/autogluon#5209]
**Bug Report Checklist**

- [x] I provided code that demonstrates a minimal reproducible example.  
- [ ] I confirmed bug exists on the latest mainline of AutoGluon via source install.  
- [ ] I confirmed bug exists on the latest stable version of AutoGluon.  

**Describe the bug**

Using different random seeds leads to identical predictions and results when fine tuning the Chronos_bolt_base model.

**Expected behavior**

I think different random seeds should give slightly different results because of randomness in selecting training samples and in generating probabilistic forecasts. 

**To Reproduce**

```
import numpy as np
import pandas as pd
from autogluon.timeseries import TimeSeriesDataFrame, TimeSeriesPredictor

data = TimeSeriesDataFrame.from_path(
    "https://autogluon.s3.amazonaws.com/datasets/timeseries/australian_electricity_subset/test.csv"
)

prediction_length = 48
train_val_data, test_data = data.train_test_split(prediction_length)
train_data, val_data = train_val_data.train_test_split(prediction_length)

seeds = [1, 2, 3]
expname = 'chronos_finetune_seed_test'
res_dfs = {}
predictions = {}

for seed in seeds:
    predictor_FT = TimeSeriesPredictor(
                        prediction_length=prediction_length, 
                        path=f"{expname}/version_{seed}",
                    )
    predictor_FT.fit(
                    train_data, 
                    tuning_data = val_data, 
                    hyperparameters={
                        "Chronos": {
                            "model_path": 'bolt_base',
                            "ag_args": {"name_suffix": "FineTuned"},
                            "fine_tune": True,
                            "fine_tune_lr": 1e-5,
                            "fine_tune_steps": 4,
                            "fine_tune_batch_size": 32,
                            "eval_during_fine_tune": True, 
                            "fine_tune_eval_max_items": None,
                            "fine_tune_trainer_kwargs": {
                                "evaluation_strategy": "steps",  
                                "eval_steps": 1,
                                "save_strategy": "steps",
                                "save_steps": 1,
                                "logging_strategy": "steps",
                                "logging_steps": 1,
                                "load_best_model_at_end": True,
                            },
                        }
                    },
                    enable_ensemble=False, 
                    random_seed=seed,
                )
    predictions[seed] = predictor_FT.predict(train_val_data, use_cache=False, random_seed=seed)
    res_dfs[seed] = predictor_FT.leaderboard(data=test_data, use_cache=False)

res_labeled = [
    df.assign(seed=seed) 
    for seed, df in res_dfs.items()
]
all_results = pd.concat(res_labeled, ignore_index=True)
pd.set_option('display.float_format', '{:.10f}'.format)
print(all_results)  # see below
print(predictions[1].equals(predictions[2]))   # output: True
print(predictions[2].equals(predictions[3]))   # output: True
```

all_results:

model | score_test | score_val | pred_time_test | pred_time_val | fit_time_marginal | fit_order | seed
-- | -- | -- | -- | -- | -- | -- | --
ChronosFineTuned[bolt_base] | -0.0368082599 | -0.0352526400 | 0.7504544258 | 0.0608270168 | 16.9453992844 | 1 | 1
ChronosFineTuned[bolt_base] | -0.0368082599 | -0.0352526400 | 0.7542641163 | 0.0673704147 | 14.3808441162 | 1 | 2
ChronosFineTuned[bolt_base] | -0.0368082599 | -0.0352526400 | 0.7540547848 | 0.0669255257 | 14.5237402916 | 1 | 3

In addition, the predictor_log.txt for each seed contains the training log for all seeds. Please find chronos_finetune_seed_test/version_1/logs/predictor_log.txt in the attachment in details. chronos_finetune_seed_test/version_2/logs/predictor_log.txt and chronos_finetune_seed_test/version_3/logs/predictor_log.txt have the same content. 

**Installed Versions**

<details>

```python
INSTALLED VERSIONS
------------------
date                : 2025-07-10
time                : 12:05:05.485874
python              : 3.10.16.final.0
OS                  : Linux
OS-release          : 4.18.0-513.18.1.el8_9.x86_64
Version             : #1 SMP Wed Feb 21 21:34:36 UTC 2024
machine             : x86_64
processor           : x86_64
num_cores           : 40
cpu_ram_mb          : 385585.890625
cuda version        : 12.565.57.01
num_gpus            : 1
gpu_ram_mb          : [22906]
avail_disk_size_mb  : 1434488

accelerate          : 1.6.0
autogluon           : None
autogluon.common    : 1.3.0
autogluon.core      : 1.3.0
autogluon.features  : 1.3.0
autogluon.tabular   : 1.3.0
autogluon.timeseries: 1.3.0
boto3               : 1.38.17
catboost            : 1.2.8
coreforecast        : 0.0.15
einops              : 0.8.1
fastai              : None
fugue               : 0.9.1
gluonts             : 0.16.0
huggingface-hub     : 0.31.1
hyperopt            : 0.2.7
imodels             : None
joblib              : 1.5.0
lightgbm            : 4.6.0
lightning           : 2.4.0
matplotlib          : 3.10.1
mlforecast          : 0.13.6
networkx            : 3.4.2
numpy               : 2.1.3
onnx                : None
onnxruntime         : None
onnxruntime-gpu     : None
optimum             : None
optimum-intel       : None
orjson              : 3.10.15
pandas              : 2.2.3
psutil              : 5.9.0
pyarrow             : 20.0.0
pytorch-lightning   : 2.4.0
ray                 : 2.44.1
requests            : 2.32.3
scikit-learn        : 1.6.1
scikit-learn-intelex: None
scipy               : 1.15.1
skl2onnx            : None
spacy               : None
statsforecast       : 2.0.1
tabpfn              : None
tensorboard         : 2.19.0
torch               : 2.6.0
tqdm                : 4.67.1
transformers        : 4.49.0
utilsforecast       : 0.2.10
xgboost             : 3.0.1

[predictor_log.txt](https://github.com/user-attachments/files/21169729/predictor_log.txt)

```

</details>

**Comment by shchur** [comment]
Hi @ziyulu-uw, thanks for the question. The Chronos-Bolt model that you use above is deterministic (it directly produces a multi-step-ahead quantile forecast), so it's not affected by the random seed. Feel free to reopen the issue if you have another questions or concerns.

**Comment by ziyu-lu** [comment]
Thanks for clarification  @shchur ! However, since the training data seems to be shuffled during fine tuning (line 526 [here](https://github.com/autogluon/autogluon/blob/master/timeseries/src/autogluon/timeseries/models/chronos/model.py#L83), I think random seeds should affect training data sampling and produce different results? 

Oddly I don't seem to have permission to reopen the issue.

**Comment by MoradLaglil** [comment]
Hi,
I'm experiencing the same issue. Do you have any updates on this?
Thank you
